fish@kriegcc/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Source of this script with a bit of explanation, see .typescript-declarations/README.md
+
+UUID="fish@kriegcc"
+
+# exit immediately if any command within the script exits with a non-zero status
+set -e
+
+echo "Running build..."
+npm run build || { echo "Build failed. Exiting."; exit 1; }
+echo "Build completed successfully."
+
+# Comment out these lines if you don't want to wipe the applet folder
+rm -rf ~/.local/share/cinnamon/applets/$UUID/
+mkdir ~/.local/share/cinnamon/applets/$UUID/
+# --------------------------------------------------
+cp -rf files/$UUID/* ~/.local/share/cinnamon/applets/$UUID/
+export DISPLAY=:0; cinnamon --replace &
```

Comment:
```suggestion
```

There is a script at the base of the repository for this: `./test-spice`

Reply:
Thanks!
The problem is that `npm run build` still needs to run before copying the applet folder.
I think, I could call the `test-spice` from this script here. Would that be an option?

Reply:
I would like to keep this line:
```
export DISPLAY=:0; cinnamon --replace &
```
This is extremely useful to spot problems and errors in console while developing.
See: https://github.com/linuxmint/cinnamon-spices-applets/blob/master/.typescript-declarations/README.md#reason-for-using-export-display0-cinnamon---replace-

Reply:
Feel free to make those changes!

Reply:
> However, I think, it can become a problem when an applet relies on that its folder name is equal to its UUID.

Correct, I believe there is a way to retrieve the actual location to address this.



> Would it be possible, that the script returns an error code on failure?

Why isn't the execution of the script halted when an error is encountered?